LICENSE.txt

```diff
@@ -1,12 +1,207 @@
-Copyright (c) 2020, Salesforce.com, Inc.
+Apache License Version 2.0
```

Comment:
copied from https://github.com/salesforce/oss-template/blob/main/LICENSE.txt , with year updated to 2025, see internal thread about relicensing